### PR TITLE
created triangle and calculated its  area

### PR DESCRIPTION
**Pull request by Muhammadali2025222**

### CODE REVIEW

File: Main1.cpp [hunk 1]
```diff
@@ -6,6 +6,7 @@
 #include "Square.cpp"
 #include "Rectangle.cpp"
 #include "Circle.cpp"
+#include "Triangle.cpp"
```
Comment (abbasshah17) [inline Main1.cpp]: Why are we including `Triangle.cpp` in our main file/function? What purpose does it serve?

**Review by abbasshah17 — APPROVED** [review]
Fix the unused inclusion of the `Triangle.cpp` class in `Main1.cpp`.

File: Main1.cpp (hunk 1 above)
Comment (Muhammadali2025222) [reply Main1.cpp]: removed all the unnecessary includes in main 1

Comment (Muhammadali2025222) [reply Main1.cpp]: https://github.com/Muhammadali2025222/C-Practice/pull/11/files check this to confirm changes

**Pull request merged**